contracts/factory/src/msg.rs

```diff
@@ -31,7 +31,7 @@ pub struct CoreParams {
     pub unbonding_safe_period: u64,
     pub unbond_batch_switch_time: u64,
     pub lsm_min_bond_amount: Uint128,
-    pub lsm_redeem_threshold: u64,
+    pub lsm_redeem_threshold: u64,    //amount of lsm denoms
```

Comment:
```suggestion
    pub lsm_redeem_threshold: u64,    // lsm denoms count
```
`amount` word may confuse into some sum of tokens or smth